tests/multigrid/dof_06.cc

```diff
@@ -0,0 +1,172 @@
+#include <deal.II/base/exceptions.h>
+
+#include <deal.II/dofs/dof_handler.h>
+
+#include <deal.II/grid/grid_generator.h>
+#include <deal.II/grid/grid_tools.h>
+#include <deal.II/grid/tria.h>
+
+#include <deal.II/multigrid/mg_constrained_dofs.h>
+
+#include <vector>
+
+constexpr int DIM = 2;
+
+using namespace dealii;
+
+template <typename FaceType>
+void
+print_face(const FaceType &face, unsigned int level)
+{
+  AssertThrow(
+    face->get_fe(0).n_dofs_per_face() > 0,
+    ExcMessage(
+      "Please use a finite element with at least 1 DoF for this test"));
+
+  std::vector<types::global_dof_index> mg_dofs(
+    face->get_fe(0).n_dofs_per_face());
+  face->get_mg_dof_indices(level, mg_dofs);
+
+  // Print face DoFs
+  std::cout << mg_dofs[0];
+  for (unsigned int i_dof = 1; i_dof < mg_dofs.size(); ++i_dof)
```

Comment:
You could also just use a range-based loop here as well:
```
for (const auto mg_dof : mg_dofs)
  deallog << ", " << mg_dof;
```

Reply:
How can i exclude the first element from such a loop?

Reply:
Ah, I missed that you want to treat the first element special. Of course, you could use newlines instead
```
for (const auto mg_dof : mg_dofs)
  deallog << mg_dof <<'\n';
```
or just leave as is. Either is fine.